spot-client/src/spot-tv/backend/SpotTVBackendService.test.js

```diff
@@ -0,0 +1,135 @@
+import { persistence } from 'common/utils';
+
+import { SpotTvBackendService } from './SpotTvBackendService';
+
+// FIXME duplicated with SpotBackendService.test.js
```

Comment:
I think the duplication with these mocks is fine. If you really wanted you could create helpers that do these mocks but I couldn't argue for them at this point, to be honest. I understand the un-easiness with copy/paste but this seems small enough to be okay (admittedly that's how problems start though)